Zend/Optimizer/zend_func_infos.h

```diff
@@ -0,0 +1,24 @@
+/* This is a generated file, edit the .stub.php files instead. */
+
+static const func_info_t func_infos[] = {
+    F1("zend_version", MAY_BE_STRING),
+    FN("func_get_args", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_ANY),
+    F1("get_class_methods", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_STRING),
+    F1("get_included_files", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_STRING),
+    F1("get_required_files", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_OF_STRING),
```

Comment:
There's no need to included aliases in the output.

Reply:
Currently, I require aliases to have the same `@refcount` as the original function, but we could simply drop this requirement if we exclude them anyway from the generated func info. Would doing so make sense? Personally, I'm fine with the current behavior.

Reply:
No strong opinion, but I'd probably drop the requirement to have `@refcount` on the alias.

Reply:
I've just dropped it.